src/sentry/monitors/consumers/clock_tick_consumer.py

```diff
@@ -28,10 +30,27 @@ def process_clock_tick(message: Message[KafkaPayload | FilteredPayload]):
     wrapper: ClockTick = MONITORS_CLOCK_TICK_CODEC.decode(message.payload.value)
     ts = datetime.fromtimestamp(wrapper["ts"], tz=timezone.utc)
 
-    logger.info("process_clock_tick", extra={"reference_datetime": str(ts)})
+    volume_anomaly_result = TickVolumeAnomolyResult.from_str(
+        wrapper.get("volume_anomaly_result", "normal")
+    )
+
+    logger.info(
+        "process_clock_tick",
+        extra={"reference_datetime": str(ts), "volume_anomaly_result": volume_anomaly_result.value},
+    )
 
     dispatch_check_missing(ts)
```

Comment:
TODO if volume anomolous modify check missing to mark unknown instead of missing (cause theres extra logic in this task)